Cargo.toml

```diff
@@ -8,3 +8,4 @@ actix-web = "0.7"
 serde = "1.0"
 serde_derive = "1.0"
 serde_json = "1.0"
```

Comment:
we can safely remove serde_json as we no longer use it
```suggestion
```

Reply:
Thanks for catching that! Has been removed